src/qt/paymentserver.cpp

```diff
@@ -642,6 +714,15 @@ void PaymentServer::fetchRequest(const QUrl& url)
     netManager->get(netRequest);
 }
 
+void PaymentServer::sendSignature(const QUrl& url)
+{
```

Comment:
Do we need to error if this fails?

Reply:
In case the request does not succeed, the user is notified.